src/Network/Network.Test/ScenarioTests/IpGroupTest.ps1

```diff
@@ -0,0 +1,70 @@
+﻿# ----------------------------------------------------------------------------------
```

Comment:
Extra file? There's already `IpGroupsTests.ps1`

Reply:
I have removed IpGroupsTests.ps1

Reply:
ps1 file should have the same name cs file does, so your test files should be called `IpGroupTests.ps1`

Reply:
This file has been removed